app/grandchallenge/blogs/filters.py

```diff
@@ -9,8 +10,15 @@ class BlogFilter(FilterSet):
     tags = ModelMultipleChoiceFilter(
         queryset=Tag.objects.all(), widget=Select2MultipleWidget, label="Tags",
     )
+    authors = ModelMultipleChoiceFilter(
+        queryset=User.objects.filter(
+            blog_authors__in=Post.objects.all()
+        ).all(),

```

Comment:
In Django you should never use `User` directly. Use `get_user_model()` instead. See https://docs.djangoproject.com/en/3.2/topics/auth/customizing/#django.contrib.auth.get_user_model. 

You can also save a query by doing `get_user_model().objects.exclude(blog_authors__isnull=True).order_by("username")`

The `order_by` is there as I think the default ordering for a django user is by signup date.